packages/ui/Touchable/tests/Touchable-specs.js

```diff
@@ -55,10 +62,10 @@ describe('Touchable', () => {
 			fireEvent.mouseDown(component, ev);
 			rerender(<Component holdConfig={holdConfig} onHold={handler} onHoldStart={() => {}} />);
 
-			setTimeout(() => {
-				expect(handler).toHaveBeenCalled();
-				done();
-			}, 20);
+			jest.runOnlyPendingTimers();
```

Comment:
I'm curious about the difference between using `advancetimersbyTime` and using `runonlypendingTimers`.

Reply:
As described in the link (https://jestjs.io/docs/timer-mocks), when `advancetimersbyTime(msToRun)` is called, all timers are advanced by msToRun milliseconds. **All pending** "macro-tasks" that have been queued via setTimeout() or setInterval(), and would be executed during this time frame, will be executed. 
In the Hold class that is the subject of this unit test, setInterval() is called, so there may be multiple timers and there may be cases where all timers cannot advance at once, so I used `runOnlyPendingTimers()` that only runs pending timers.